### DISCUSSION TITLE
Im complete noob how to install?

### DISCUSSION THREAD

**Discussion by Curbi6969** [tsightler/ring-mqtt#805] [Support Q&A]
So i installed mqtt and your addon also go2rtc. but when opening go2rtc there is no stream, everything ius empty.

I logged in with your addon.

Also there is no video or just a full guide step by step how to set it up and end up with the livestream on your dashboard

**Comment by tsightler** [comment]
I feel like the [camera streaming documentation](https://github.com/tsightler/ring-mqtt/wiki/Video-Streaming#home-assistant-generic-camera-configuration) is pretty step-by-step, however, this project does require users to have basic knowledge of protocols and generic camera setup in HA.  There's plenty of documentation on this on the HA website and forums around setting up generic cameras, there's nothing specific to ring-mqtt there other than what is documented.
